### PR TITLE
Moenchfnum

### PR DESCRIPTION
**Pull request by thattil**
- this PR is for both ctb and moench
- set frame number for next acquisition 
- adjust in virtual servers for the fake frame numbers that can be set
- adjust in 1g udp interface to allow to set frame numbers
- set default starting number at server start up

### CODE REVIEW

File: RELEASE.txt [hunk 1]
```diff
@@ -26,7 +26,11 @@ This document describes the differences between v6.1.0 and v6.0.0.
 - refactoring (rxr)
 - fixed patsetbit and patsetmask for moench
 - changed default vref of adc9257 to 2V for moench (from 1.33V)
+<<<<<<< HEAD
```
Comment (erikfrojdh) [inline RELEASE.txt:29]: I guess this should not be in the commit? Looks like the traces of a merge conflict?

File: slsDetectorServers/ctbDetectorServer/slsDetectorFunctionList.c [hunk 2]
```diff
@@ -898,6 +898,24 @@ int getReadoutMode() {
...
+    setU64BitReg(value, NEXT_FRAME_NUMB_LOCAL_LSB_REG,
+                 NEXT_FRAME_NUMB_LOCAL_MSB_REG);
+#ifndef VIRTUAL
+    // for 1g udp interface
+    setUDPFrameNumber(value);
+#endif
+    return OK;
+}
+
+int getNextFrameNumber(uint64_t *retval) {
```
Comment (erikfrojdh) [inline slsDetectorServers/ctbDetectorServer/slsDetectorFunctionList.c:913]: What will the virtual server return?

Comment (thattil) [reply slsDetectorServers/ctbDetectorServer/slsDetectorFunctionList.c:913]: what ever is in this register.. it is set up to be 1 at startup..

File: RELEASE.txt (hunk 1 above)
Comment (thattil) [reply RELEASE.txt:29]: Yes it does. thanks for seeing that. ill fix it soon

File: slsDetectorServers/ctbDetectorServer/slsDetectorFunctionList.c (hunk 2 above)
Comment (thattil) [reply slsDetectorServers/ctbDetectorServer/slsDetectorFunctionList.c:913]: the return is a ok or fail (useful only for eiger).. but the argument will return whatever is in the register for the next frame number.

**Pull request merged**